bitools/tableau/README.md

```diff
@@ -0,0 +1,10 @@
+Tableau Resources
+=======================
+
+#### Description
+These files provide the needed configuration to connect to HPCC from Tableau. The hpcc-jdbc.tdc file specifies the basic functionality and driver type for the jdbc connection to HPCC. The example.properties file is a template for a connnection properties file; this template file should be used to create a new properties for each HPCC cluster that you Tableau will connect to. These connection files can be placed anywhere on your system.
+
+hpcc-jdbc.tdc should be placed in the `$DOCUMENTS\My Tableau Repository\Datasources` directory. 
+
+#### Connecting to HPCC via JDBC
+Install the built jdbc jar from this project into the `$Tableau/Drivers` directory. Start Tableau select: Other Databases (JDBC). In the pop-up window type `jdbc:hpcc` into the URL field, under dialect select MySQL, provide your HPCC credentials, and select the properties file you created above for the HPCC cluster you wish to connect to.
```

Comment:
Nitpick, does it make sense to reference the jar by it's official name? or the HPCC jdbcdriver.jar ?
Also, can we split this into a bulleted list?